### ISSUE TITLE
Cannot recognize JSON with many Cyrillic characters

### ISSUE THREAD

**Issue by FranklinYu** [NetSPI/JSONBeautifier#6]
https://unicode-table.com/en/#cyrillic

Try adding `ЀЀЀЀЀЀ` into the JSON response and the "JSON Beautifier" tab won't show.

**Comment by JacobReynolds** [comment]
Burp won't allow me to paste those characters into the response windows. Did you configure specific unicode settings to allow it?

**Comment by FranklinYu** [comment]
No, I don't think there is such settings. Burp itself is terrible at Unicode. I simply intercept some JSON response and add those characters. I suspect that pasting Unicode isn't supported either, so I paste Base64 encoded text (`0IDQgNCA0IDQgNCA`), and hit `Ctrl` + `Shift` + `B` to decode it in proxy intercept tab. Neither copy nor paste of Unicode works in Burp.

This bug only happens to HTTP response, according to my experiments.
